immutablecollections/_immutableset.py

```diff
@@ -92,6 +98,10 @@ def immutableset(
         if value not in containment_set:
             containment_set.add(value)
             iteration_order.append(value)
+        elif forbid_duplicate_elements:
+            raise ValueError(
+                "Input collection has duplicate items and forbid_duplicate_elements=False"
```

Comment:
This got merged before I could take a look, but could this be modified to give the `repr` of the duplicate element? This would make debugging much easier.